src/main.js

```diff
@@ -313,15 +313,15 @@ export default class FeatureService {
     return true
   }
 
-  _getServiceMetadata () {
+  _getServiceMetadata() {
     if (this.serviceMetadata !== null) return Promise.resolve(this.serviceMetadata)
 
     const params = new URLSearchParams({f: 'json'})
 
     this._appendTokenIfExists(params)
 
     return new Promise((resolve, reject) => {
-      fetch(`${this._esriServiceOptions.url}?${params.toString()}`)
+      fetch(`${this._esriServiceOptions.url}?${params.toString()}`, this._esriServiceOptions.fetchOptions)
```

Comment:
This whole line should probably be replaced with a call to `this._requestJson` which is my bad from the original code

Reply:
happy to swap that out